example/jobflow/JobFlow.yaml

```diff
@@ -5,8 +5,12 @@ metadata:
   namespace: default
 spec:
   jobRetainPolicy: delete   # After jobflow runs, keep the generated job. Otherwise, delete it.
+  maxRetry: 1               # If maxRetry not defined, use this global maxRetry count. 
   flows:
     - name: a
+      patch: 
```

Comment:
What is the function of patch

Reply:
When the maxRetry of task a is different from the global setting, the patch method is supported.

Reply:
Yes, I mean, I don't know the necessity of this field. It seems to be fine without it. What do you think?

Reply:
Hmmm... 🤔️ Thanks @hwdef 
In my usage scenario, I still need to Patch some information to replace the content of the JobTemplate. Avoid partial differences causing an explosion in the number of JobTemplates

Reply:
> Yes, I mean, I don't know the necessity of this field. It seems to be fine without it. What do you think?

+1, I also think that this field is unnecessary. Or you can reflect the purpose of this field in the design doc (code logic design), which will make it easier for everyone to understand.

Reply:
> > Yes, I mean, I don't know the necessity of this field. It seems to be fine without it. What do you think?
> 
> +1, I also think that this field is unnecessary. Or you can reflect the purpose of this field in the design doc (code logic design), which will make it easier for everyone to understand.

Thanks @googs1025 
One User case:
```yaml
apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  name: a
spec:
  tasks:
    - spec:
          containers:
            - image: tf2.16-cuda11-xxx:v1.0.1
              command:
                - sh
                - -c
                - ./script.py --dateset a.txt --mode 1
              imagePullPolicy: IfNotPresent
              name: nginx
              resources:
                requests:
                  cpu: "1"
          restartPolicy: OnFailure
---
apiVersion: flow.volcano.sh/v1alpha1
kind: JobFlow
metadata:
  name: test-flow1
  namespace: default
spec:
  jobRetainPolicy: delete
  flows:
    - name: a
---
apiVersion: flow.volcano.sh/v1alpha1
kind: JobFlow
metadata:
  name: test-flow2
  namespace: default
spec:
  jobRetainPolicy: delete
    - name: a
      patch:
          spec:
             tasks:
               - spec:
                     containers:
                        - image: tf2.16-cuda11-xxx:v1.0.2 # just different image
                           command:
                              - sh
                              - -c
                              - ./script.py --dateset b.txt --mode 2 # just different args
```

Reply:
I am more concerned about the role of patch and whether this patch is necessary because it will make cr more complicated. The patch field alone cannot let users know what the role is.